BasicRPG/BasicRPG/Program.cs

```diff
@@ -1,160 +1,121 @@
-﻿using System.Runtime.CompilerServices;
-using System.Security.Cryptography.X509Certificates;
-
-class Program
+﻿class Program 
 {
-    static void Main(string[] args)
+    static void Main(string[] args) 
     {
         Console.WriteLine(" --------------------- \nWelcome to the gladiator trials!\n-----------------------");
-        Console.WriteLine("Please choose a class !");
-        Character player = characterSelection();
-        Console.WriteLine("Time to BATTLE !");
-        Enemy enemy = new Enemy();
-        battle(player, enemy);
+        Console.WriteLine("Please choose a class!");
+        Character player = CharacterSelection();
+        Console.WriteLine("Time to Battle!");
+        Character enemy = new Character("Tyler1");
```

Comment:
Changed `Enemy` class to `Character` since I deleted the `Enemy` class as it was redundant.